components/contract-functions/contract-function.tsx

```diff
@@ -149,25 +159,25 @@ export const ContractFunctionsPanel: React.FC<ContractFunctionsPanelProps> = ({
           pr={{ base: 0, md: 3 }}
           mb={{ base: 3, md: 0 }}
         >
-          {sortedFunctions.map((fn) => (
-            <ListItem key={fn.signature} my={0.5}>
+          {fnsOrEvents.map((fn) => (
+            <ListItem key={fn.name} my={0.5}>
```

Comment:
`name` is not good enough as a key, there are functions that may have the same name which will then clash

Reply:
right but events don't have signature, just makes it annoying then to use the key

Reply:
can u do: `key={fn?.signature || fn.name}` ? would solve that problem, no?

Reply:
@nachoiacovino this part is still a blocker

Reply:
can't do that but I'll do the check